PocketKit/Sources/PocketKit/Premium/PremiumUpgradeView.swift

```diff
@@ -175,12 +176,51 @@ Refunds are not available for unused portions of a subscription.
 }
 
 private struct PremiumTermsView: View {
+    @State var showPrivacyPolicy = false
+    @State var showTermsOfService = false
+
     var body: some View {
         HStack(spacing: 16) {
-            Button { } label: { Text("Privacy Policy").style(.terms) }
-            Button { } label: { Text("Terms of Service").style(.terms) }
+            Button(action: {
+                self.showPrivacyPolicy = true
+            }, label: { Text("Privacy Policy").style(.terms) })
+            .sheet(isPresented: $showPrivacyPolicy) {
+                if let privacyUrl = getUrlFor(typeOf: .PrivacyPolicy) {
+                    SFSafariView(url: privacyUrl)
+                }
+            }
+            .accessibilityIdentifier("privacy-policy")
+            Button(action: {
+                self.showTermsOfService = true
+            }, label: { Text("Terms of Service").style(.terms) })
+            .sheet(isPresented: $showTermsOfService) {
+                if let toSUrl = getUrlFor(typeOf: .TermsOfService) {
+                    SFSafariView(url: toSUrl)
+                }
+            }
+            .accessibilityIdentifier("terms-of-service")
```

Comment:
since we have accessibility identifiers, will we also be writing tests?

Reply:
as discussed prior in other meetings, since settings is mostly all redirected links, there's no need to write tests for settings screen.